test/test_functional.py

```diff
@@ -103,6 +103,25 @@ def test_griffinlim(self):
 
         self.assertTrue(torch.allclose(ta_out, lr_out, atol=5e-5))
 
+    def test_batch_griffinlim(self):
+
+        torch.random.manual_seed(42)
+        tensor = torch.rand((1, 201, 6))
+
+        n_fft = 400
```

Comment:
Are these parameters special to the batch test?

Reply:
Using the same as `test_torchscript_spectrogram`, but we can change them or we could set them in the class directly.